alphastats/gui/pages/02_Import Data.py

```diff
@@ -29,26 +29,26 @@
 
 runtime = get_instance()
 session_id = get_script_run_ctx().session_id
-session_info = runtime._session_mgr.get_session_info(session_id)
+# session_info = runtime._session_mgr.get_session_info(session_id)
 
 user_session_id = session_id
-st.session_state["user_session_id"] = user_session_id
+st.session_state[SessionStateKeys.USER_SESSION_ID] = user_session_id
 
 if "loader" not in st.session_state:
-    st.session_state["loader"] = None
+    st.session_state[SessionStateKeys.LOADER] = None
 
 if "gene_to_prot_id" not in st.session_state:
-    st.session_state["gene_to_prot_id"] = {}
+    st.session_state[SessionStateKeys.GENE_TO_PROT_ID] = {}
 
 if "organism" not in st.session_state:
-    st.session_state["organism"] = 9606  # human
+    st.session_state[SessionStateKeys.ORGANISM] = 9606  # human
 
 
 def load_options():
     from alphastats.gui.utils.options import plotting_options, statistic_options
 
-    st.session_state["plotting_options"] = plotting_options
-    st.session_state["statistic_options"] = statistic_options
+    st.session_state[SessionStateKeys.PLOTTING_OPTIONS] = plotting_options
```

Comment:
high level view: there is _a lot_ happening with the session state already only on this page .. maybe we should also revisit its use and try to limit it as much as possible

Reply:
Once I am done with the bookkeeping and the page tests that's exactly what I plan to do, also to make the saving and loading of settings more straighforward.